sources/install/package_ad.sh

```diff
@@ -1366,6 +1366,20 @@ function install_sccmwtf() {
     add-to-list "sccmwtf,https://github.com/xpn/sccmwtf,This code is designed for exploring SCCM in a lab."
 }
 
+function install_smbclientng() {
+    colorecho "Installing smbclientng"
+    git -C /opt/tools/ clone --depth 1 https://github.com/p0dalirius/smbclientng /opt/tools/smbclientng/
```

Comment:
The repo doesn't exists with this URL : https://github.com/p0dalirius/smbclientng

https://github.com/p0dalirius/smbclient-ng

Reply:
Fixed